classes/SpecificPrice.php

```diff
@@ -34,6 +34,7 @@ class SpecificPriceCore extends ObjectModel
     public $id_shop_group;
     public $id_currency;
     public $id_country;
+    public $id_zone;
```

Comment:
Is snake_case mandatory here ? If not, please use camelCase.

Reply:
I must leave snake_case because otherwise I would change the table too